UnityProject/Assets/Scripts/Chat/ChatRelay.cs

```diff
@@ -142,16 +150,26 @@ private void UpdateClientChat(string message, ChatChannel channels)
 			name = "<b>[" + channels + "]</b> ";
 		}
 
-		if ((PlayerManager.LocalPlayerScript.GetAvailableChannelsMask(false) & channels) == channels && (chatEvent.channels & channels) == channels) {
-            //Chatevent UI entry:
+		ChatChannel checkChannels;
+		if (PlayerManager.LocalPlayerScript == null)
```

Comment:
Can make this into a ternary statement

Reply:
sometimes ternary ones are less readable or end up too long, so I think it's fine here as is

Reply:
I opted for the if statements so the lines didn't get too long and cluttered